src/actions/serviceAccounts.js

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { getServiceAccounts } from '../api';
+import { getSelectedNamespace } from '../reducers';
+
+export function fetchServiceAccountsSuccess(data, namespace) {
```

Comment:
We've made some changes recently to the action creators and reducers. These changes should also be made for the new serviceaccounts functionality, both for consistency, and to ensure correct behaviour.

- changes to support loading all namespaces https://github.com/tektoncd/dashboard/pull/170
  - remove `namespace` from the success action creators
  - update reducers to read namespace value from the resource metadata
- allow overriding the selected namespace https://github.com/tektoncd/dashboard/pull/164
  - update 'fetch' action creators to accept namespace as a parameter (for use by extensions)
- expose action creators to extensions (where appropriate)
  - https://github.com/tektoncd/dashboard/blob/master/src/containers/Extension/actions.js

Reply:
Thanks for the feedback Alan. I'll look over these and make the changes.